src/Surfnet/ServiceProviderDashboard/Infrastructure/Manage/Dto/ManageEntity.php

```diff
@@ -167,10 +167,18 @@ public function isOidcngResourceServer()
         return false;
     }
 
+    public function isExcludedFromPushSet()
+    {
+        if (is_null($this->getMetaData()->getCoin()->getExcludeFromPush())) {
+            return false;
+        }
+        return true;
+    }
+
     public function isExcludedFromPush()
     {
         if (is_null($this->getMetaData()->getCoin()->getExcludeFromPush())) {
-            return true;
+            return false;
```

Comment:
Caught a bug, where the non-existence of the coin was misinterpreted to be excluded from push. Where this is the other way around..